edit/buildozer.go

```diff
@@ -417,6 +417,46 @@ func cmdRemove(opts *Options, env CmdEnvironment) (*build.File, error) {
 	return nil, nil
 }
 
+func cmdRemoveIfEqual(opts *Options, env CmdEnvironment) (*build.File, error) {
+	attr := env.Args[0]
+	args := env.Args[1:]
+	expr := getAttrValueExpr(attr, args, env)
+	cmp, err := func() (func(build.Expr) bool, error) {
```

Comment:
Why is it in a function? Might be more readable to remove this function which is executed immediately.

Reply:
I tend to find immediately invoked functions, and a single assignment more clear.  I tend to write more C++ where that allows the variable to be const, but that isn't an advantage here.  Thanks for pointing that out; removed.